src/Castle.Core.Tests/Castle.Core.Tests.csproj

```diff
@@ -20,7 +20,13 @@
 		<StartupObject>Program</StartupObject>
 	</PropertyGroup>
 
-	<ItemGroup Condition="'$(TargetFramework)'=='netcoreapp1.1'">
+	<ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.0'">
+		<Compile Remove="Components.DictionaryAdapter.Tests\Xml\**" />
+		<Compile Remove="Services.Logging.Tests\log4netIntegration\**" />
+		<Compile Remove="Services.Logging.Tests\NLogIntegration\**" />
+	</ItemGroup>
+
+	<ItemGroup Condition="'$(TargetFramework)'=='netcoreapp3.1'">
```

Comment:
(There's some avoidable duplication here, you could have written the condition as follows:)

```csharp
<ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.0'OR'$(TargetFramework)'=='netcoreapp3.1'">`
```

(It would've been more consistent to exclude these code files using conditional compilation symbols `FEATURE_TEST_LOG4NETINTEGRATION` and `FEATURE_TEST_NLOGINTEGRATION`&mdash;like we have for Serilog.)

(All of that said, feel free to improve this, but I don't particularly mind if it's left as is for now, as I'll likely work on these code lines in #495 again anyway.)